conans/test/functional/imports_tests.py

```diff
@@ -31,6 +32,29 @@ def _set_up(self):
         client.run("export lasote/testing")
         return client
 
+    def repackage_test(self):
+        client = self._set_up()
+        conanfile = """from conans import ConanFile
+class TestConan(ConanFile):
+    requires='LibC/0.1@lasote/testing'
+    def imports(self):
+        self.copy("license*", dst="licenses", folder=True, ignore_case=True)
+        self.keep_imports = True
```

Comment:
Maybe is easier to explain or at least more homogeneous to use an attribute class to activate it? Similar to no_copy_source etc. Instead of declaring it in the imports method.